Format reducer.cpp, hpp

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #58603 [Reducer] Remove replica size == 1 checks
* #58595 [DDP] Remove train call to module copies
* #58594 [Reducer] move comment to the right place
* **#58593 Format reducer.cpp, hpp**
* #58592 [Reducer] Completely remove VariableIndex

Per title

Differential Revision: [D28528465](https://our.internmc.facebook.com/intern/diff/D28528465/)

rohan-varma committed:
Format reducer.cpp, hpp

Per title

Differential Revision: [D28528465](https://our.internmc.facebook.com/intern/diff/D28528465/)

[ghstack-poisoned]

rohan-varma committed:
Update on "Format reducer.cpp, hpp"


Per title

Differential Revision: [D28528465](https://our.internmc.facebook.com/intern/diff/D28528465/)

[ghstack-poisoned]

zhaojuanmao reviewed (approved):
clang format happened once a while:(

rohan-varma committed:
Update on "Format reducer.cpp, hpp"


Per title

Differential Revision: [D28528465](https://our.internmc.facebook.com/intern/diff/D28528465/)

[ghstack-poisoned]

rohan-varma committed:
Update on "Format reducer.cpp, hpp"


Per title

Differential Revision: [D28528465](https://our.internmc.facebook.com/intern/diff/D28528465/)

[ghstack-poisoned]
